core/src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -111,7 +115,7 @@ object RawlsBillingProjectResponse {
     project.status,
     project.message,
     project.azureManagedAppCoordinates,
-    project.azureManagedAppCoordinates.map(_ => CloudPlatform.AZURE).getOrElse(CloudPlatform.GCP).toString
+    platform.map(_.toString).getOrElse("Unknown")
```

Comment:
Can you update the enum in the Swagger documentation to indicate that "Unknown" is a possibility? We may want all caps for consistency.